mfr/extensions/pdf/render.py

```diff
@@ -25,7 +26,7 @@ def render(self):
             logger.debug('Extension not found in supported list!')
             return self.TEMPLATE.render(
                 base=self.assets_url,
-                url=download_url.geturl(),
+                url=re.sub(r'\'', '\\\'', download_url.geturl()),
```

Comment:
This is fine, because the quotation mark is the only charachter we need to change - It's very specifically only single quotation marks that need to be represented the way they should be represented in a single quote string. (so it doesn't conflict with the js string's quote that this is printed inside of.) If a url has `\` in it, it's not a valid url (https://tools.ietf.org/html/rfc3986#section-2)

Reply:
Use String.raw``

Reply:
Note this is a cheap messy fix